apps/xomg_tasks/mix.exs

```diff
@@ -0,0 +1,26 @@
+defmodule OMG.XomgTasks.MixProject do
+  @moduledoc """
+  This is just a proxy app to hold and use all the code related to running `xomg` Mix.Tasks.
+
+  NOTE: this is not a proper mix app, just some Mix.Tasks which calls into other mix apps
+  """
+  use Mix.Project
+
+  def project do
+    [
+      app: :xomg_tasks,
+      version: OMG.Umbrella.MixProject.umbrella_version(),
+      build_path: "../../_build",
+      deps_path: "../../deps",
+      lockfile: "../../mix.lock",
+      elixir: "~> 1.6",
```

Comment:
higher elixir version

Reply:
it works with 1.6 (just like the rest of subapps), so why?

Reply:
rest of project work with 1.8,
In the future could be some issue with that